pkg/prm/config.go

```diff
@@ -4,26 +4,30 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
+	"time"
 
 	"github.com/Masterminds/semver"
 	"github.com/rs/zerolog/log"
 	"github.com/spf13/viper"
 )
 
 const (
-	PuppetCmdFlag    string      = "puppet"
-	BackendCmdFlag   string      = "backend"
-	PuppetVerCfgKey  string      = "puppetversion" // Should match Config struct key.
-	BackendCfgKey    string      = "backend"       // Should match Config struct key.
-	DefaultPuppetVer string      = "7"
-	DefaultBackend   BackendType = DOCKER
-	ToolPathCfgKey   string      = "toolpath"
+	PuppetCmdFlag      string      = "puppet"
+	BackendCmdFlag     string      = "backend"
+	PuppetVerCfgKey    string      = "puppetversion" // Should match Config struct key.
+	BackendCfgKey      string      = "backend"       // Should match Config struct key.
+	DefaultPuppetVer   string      = "7"
+	DefaultBackend     BackendType = DOCKER
+	DefaultToolTimeout int         = 1800 // 30 minutes
+	ToolPathCfgKey     string      = "toolpath"
+	ToolTimeoutCfgKey  string      = "toolTimeout"
```

Comment:
Minor reorder for readability, set the default after the key

```suggestion
	ToolPathCfgKey     string      = "toolpath"
	ToolTimeoutCfgKey  string      = "toolTimeout"
	DefaultToolTimeout int         = 1800 // 30 minutes
```

Reply:
Done